### ISSUE TITLE
Unexpected Exit when using lego_brush_image configuration

### ISSUE THREAD

**Issue by vivianxjw** [windingwind/seal-3d#9]
I am facing an issue when running the model with the lego_brush_image or lego_anchor configurations. The training process starts, but the program automatically exits after loading the configuration and model. However, it works fine when using lego_bbox configuration.

![Image](https://github.com/user-attachments/assets/9aec420a-ce18-4e4d-8473-f7640457b70f)

**Comment by windingwind** [comment]
I didn't see any errors in the log above. From which line does the execution exit?

**Comment by vivianxjw** [comment]
This is my input：
(torch-ngp) D:\seal>python main_SealNeRF.py data/nerf_synthetic/lego/ --workspace exps/lego_ngp_bbox -O --bound 1.0 --scale 0.8 --dt_gamma 0 --pretraining_epochs 100 --extra_epochs 150 --pretraining_local_point_step 0.005 --pretraining_surrounding_point_step -1 --pretraining_lr 0.05 --ckpt exps/lego_ngp/checkpoints/ngp_ep0150.pth --teacher_workspace exps/lego_ngp --teacher_ckpt exps/lego_ngp/checkpoints/ngp_ep0150.pth --seal_config data/seal/lego_brush_imagesquare/ --eval_interval 100 --eval_count 10

It only displays these messages, and then exits without training.

Namespace(path='data/nerf_synthetic/lego/', O=True, test=False, workspace='exps/lego_ngp_bbox', seed=0, iters=30000, extra_epochs=150, lr=0.01, ckpt='exps/lego_ngp/checkpoints/ngp_ep0150.pth', num_rays=4096, log2_hashmap_size=19, cuda_ray=True, max_steps=1024, num_steps=512, upsample_steps=0, update_extra_interval=16, max_ray_batch=4096, patch_size=1, fp16=True, ff=False, tcnn=False, color_space='srgb', preload=True, bound=1.0, scale=0.8, offset=[0, 0, 0], dt_gamma=0.0, dt_gamma_proxy=0.0078125, min_near=0.2, density_thresh=10, bg_radius=-1, seal_config='data/seal/lego_brush_imagesquare/', pretraining_epochs=100, pretraining_local_point_step=0.005, pretraining_local_angle_step=45, pretraining_surrounding_point_step=-1.0, pretraining_surrounding_angle_step=45, pretraining_surrounding_bounds_extend=0.1, pretraining_global_point_step=-1, pretraining_global_angle_step=45, pretraining_batch_size=6144000, pretraining_lr=0.05, custom_pose=False, gui=False, W=1920, H=1080, radius=5, fovy=50, max_spp=64, error_map=False, clip_text='', rand_pose=-1, teacher_workspace='exps/lego_ngp', teacher_ckpt='exps/lego_ngp/checkpoints/ngp_ep0150.pth', secondary_teacher_workspace=None, secondary_teacher_ckpt='latest', secondary_teacher_options={}, eval_interval=100, eval_count=10, test_type='test', proxy_batch=1)

I found that the issue is with Plane.best_fit. I can successfully obtain the plane in the environment, but it doesn't work properly during training.

![Image](https://github.com/user-attachments/assets/6aa131f4-bcce-4f80-9d21-354fea8d2361)

**Comment by windingwind** [comment]
If this is the case, you'll need to make sure the input point cloud forms a plane.

**Comment by vivianxjw** [comment]
I solved the problem by downgrading the library version. Thank you so much!
